collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -251,11 +252,34 @@ public void onCreate() {
 
         if (BuildConfig.BUILD_TYPE.equals("odkCollectRelease")) {
             Timber.plant(new CrashReportingTree());
-        } else {
+        } else if (!BuildConfig.BUILD_TYPE.equals("debug")) {
```

Comment:
Why `!BuildConfig.BUILD_TYPE.equals("debug")` ?

Reply:
So that if someone forks the repo and wants to see the debug logs, then it works but doesn't enable strict mode as well which might not be useful to others

Reply:
But we have three build variants:
odkCollectRelease - our releases
release - fork releases 
debug

so, in this case, it's enabled in `release` but should be in `debug`.

Reply:
It is only enabled in `debug`. The `robolectric` build check is nested within DEBUG block.

odkCollectRelease - strict mode disabled
release - strict mode disabled
debug - strict mode enabled

Reply:
But in your if statement you have `not equals BuildConfig.BUILD_TYPE`

Reply:
Oops, my bad. :grin:

Reply:
Fixed